### PR TITLE
app-admin/calamares: fix some issues

### PR DESCRIPTION
**Pull request by matijaskala**
- add missing dependencies
- fix locale configuration with openrc

### CODE REVIEW

**Comment by johu** [comment]
Could you please rebase onto current master?

File: app-admin/calamares/calamares-3.2.2.ebuild [hunk 1]
```diff
@@ -78,6 +78,8 @@ src_configure() {
 	kde5_src_configure
 	sed -i -e 's:pkexec /usr/bin/calamares:calamares-pkexec:' "${S}"/calamares.desktop
 	sed -i -e 's:Icon=calamares:Icon=drive-harddisk:' "${S}"/calamares.desktop
+	sed -i -e 's:etc/default:etc/env.d:' "${S}"/src/modules/localecfg/main.py
+	sed -i -e 's:"locale":"02locale":' "${S}"/src/modules/localecfg/main.py
```
[the remark this answers is not in the document]
Comment (johu) [reply app-admin/calamares/calamares-3.2.2.ebuild]: @mudler What do you think?

**Comment by a17r** [comment]
ping

**Comment by juippis** [comment]
Hey all, is this still relevant?

**Comment by wimmuskee** [comment]
Well, 3.2.2 got removed from the tree with https://github.com/gentoo/gentoo/commit/61ed2b3ed7b15d05ac2e83144d62ed3de24a0dc2

**Comment by matijaskala** [comment]
Anyone wanting to use calamares with an openrc-based system can take a look at [my collection of patches](https://github.com/matijaskala/gentoo-patches/tree/master/app-admin/calamares). Closing this PR.

**Pull request closed without merging**